track/templates/en/domains.html

```diff
@@ -28,8 +28,9 @@
 	          <th class="never"></th>
 	          <th class="all">Domain</th>
 	          <th class="never">Agency</th>
-	          <th class="min-tablet">Enforces HTTPS</th>
-	          <th class="min-tablet">HSTS</th>
+	          <th class="min-tablet">IT PIN Compliant</th>
```

Comment:
ITPIN, not IT PIN
https://www.canada.ca/en/treasury-board-secretariat/services/information-technology/policy-implementation-notices/direction-electronic-data-residency.html  <-- example